lib/htmlbars-plugin/index.js

```diff
@@ -0,0 +1,123 @@
+function ClassMacroTransform() {
+  this.syntax = null;
+}
+
+ClassMacroTransform.prototype.transform = function (ast) {
```

Comment:
This transform is similar to the Babel transform, but it replaces class declarations with a helper. So, something like this:

```hbs
<div class="@a">Hi</div>
{{#my-component class="@b"}}There{{/my-component}}
```

...would become...

```hbs
<div class={{classify "a"}}>Hi</div>
{{#my-component class=(classify "b")}}There{{/my-component}}

```